scheduler/src/cook/mesos/api.clj

```diff
@@ -740,6 +766,13 @@
                   "success" "failed")
                 :job.state/running "running"
                 :job.state/waiting "waiting")
+        constraints (->> job
+                         :job/constraint
+                         (map util/remove-datomic-namespacing)
+                         (map (fn [{:keys [attribute operator pattern]}]
+                                (->> [attribute (clojure.string/upper-case (name (util/without-ns operator))) pattern]
+                                     (remove nil?)
```

Comment:
Why can we have `nil` values in the constraint? Doesn't the schema protect against that?